core/src/main/java/org/apache/iceberg/CachingCatalog.java

```diff
@@ -30,23 +35,98 @@
 import org.apache.iceberg.catalog.TableIdentifier;
 import org.apache.iceberg.exceptions.AlreadyExistsException;
 import org.apache.iceberg.relocated.com.google.common.collect.ImmutableList;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
+/**
+ * Class that wraps an Iceberg Catalog to cache table's.
+ *
+ * When `expirationIntervalMillis` is non-zero, the cache will expire tables after
```

Comment:
Javadoc doesn't support markdown syntax, so if you want to use fixed-width type you have to wrap it like this: `{@code expirationIntervalMillis}`.

Reply:
Fixed.